src/olympia/scanners/models.py

```diff
@@ -33,6 +39,10 @@ class ScannerResult(ModelBase):
     matched_rules = models.ManyToManyField(
         'ScannerRule', through='ScannerMatch'
     )
+    state = models.PositiveSmallIntegerField(
```

Comment:
If it was a NullBooleanField instead, you'd get a fancy green/red tick for free instead of the quite verbose text in the admin (which you can replicate manually by using a method + setting `<methodname>.boolean = True`, but might as well use a true boolean field)

Maybe it's worth exploring, unless you think we're going to get more states than just `true`/`false`/`null` ?

Reply:
I thought about using a nullable boolean field but @wagnerand and I discussed and we thought that we might have more states in the future.